misoc/cores/spi_flash.py

```diff
@@ -59,8 +69,13 @@ def __init__(self, pads, dummy=15, div=2, with_bitbang=True):
 
         pads.cs_n.reset = 1
 
-        dq = TSTriple(spi_width)
-        self.specials.dq = dq.get_tristate(pads.dq)
+        if spi_width > 1:
+            dq = TSTriple(spi_width)
+            self.specials.dq = dq.get_tristate(pads.dq)
+        else:
+            dq = SpiIF(pads.miso, pads.mosi, Signal())
+            assert with_bitbang == False, \
+                "Bitbang not supported with 1-bit SPI flash."
```

Comment:
Why not?

Reply:
Because it was not  working out of the box because of this line:
https://github.com/m-labs/misoc/blob/master/misoc/cores/spi_flash.py#L85
so I didn't implement it.
If you can explain the code, I can try to implement it.